tietokanta/src/main/resources/db/migration/R__Lukon_asettaminen.sql

```diff
@@ -1,7 +1,9 @@
-CREATE OR REPLACE FUNCTION aseta_lukko(tarkistettava_tunniste VARCHAR(30), uusi_lukko CHAR(36), aikaraja BIGINT)
+DROP FUNCTION aseta_lukko(tarkistettava_tunniste VARCHAR(30), uusi_lukko CHAR(36), aikaraja BIGINT );
```

Comment:
Koodipesään oli vahingossa jäänyt kahdella eri signaturella oleva aseta_lukko funktio. Näistä toisessa ei ollut riviä LOCK TABLE lukko IN ACCESS EXCLUSIVE MODE; mikä on voinut sotkea lukon toimintaa (toinen node onkin jäänyt odottamaan eikä sitä ole ohjattu pois yrittämästä). 

Siksi nyt R-migraation alussa poistetaan kilpaileva ja "väärä" funktio, ja palautetaan uusimpaan funktioon signatureksi (TEXT, CHAR 36, INT)